unit_test/CMakeLists.txt

```diff
@@ -130,7 +143,7 @@ macro(Picasso_add_tests)
         file(APPEND ${_file} "#include <tst${_test}.hpp>\n")
         set(_target ${_test}_test_${_device})
         add_executable(${_target} ${_file} ${PICASSO_UNIT_TEST_MAIN})
-        target_include_directories(${_target} PUBLIC ${_dir})
+        target_include_directories(${_target} PUBLIC ${_dir} ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Why did you add these include dirs?

Reply:
I had moved them from above. I removed BINARY, but SOURCE is needed is needed for the `TestDEVICE.cpp` files

Reply:
It's for finding `Test<BACKEND>_Category` headers in the `tst<TEST>_<BACKEND>` files created on the fly